__tests__/html/styleOptions.deprecated.newMessageButtonFontSize.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
```

Comment:
Do you want to check for warning messages for these kinds of tests?

Reply:
Leave comments / reference on when these tests should be deleted?

Reply:
Will leave comment on date.

There is an expectation at the end of the file to say we must show a warning, `{ expectDeprecations: true }`.